api/src/Repository/UserRepository.php

```diff
@@ -34,9 +34,30 @@ public function upgradePassword(UserInterface $user, string $newEncodedPassword)
         $this->_em->flush();
     }
 
-    public function filterByUser(QueryBuilder $queryBuilder, User $user): void {
-        $rootAlias = $queryBuilder->getRootAliases()[0];
-        $queryBuilder->andWhere("{$rootAlias}.id = :current_user");
-        $queryBuilder->setParameter('current_user', $user->getId());
+    /**
+     * @throws \Doctrine\ORM\NonUniqueResultException
+     * @throws \Doctrine\ORM\NoResultException
+     */
+    public function areUsersRelatedViaCamp(User $user1, User $user2): bool {
```

Comment:
Ich habe den check in der DB implementiert, weil es sein könnte dass mit den CampCollaboration::getCamp und den Camp::getCollaborations zuviele queries auf die DB gehen.